src/pages/Card/Card.page.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+    width: 22%;
+    border-style: solid;
+    margin: 1em;
+    border-radius: 1em;
+    display: flex;
+    justify-content: space-between;
+    float: left;
+    background-color: #d47b7b;
+    word-wrap: break-word;
+`;
+
+function Card({ title, description, image, width, height }) {
+    return (
+        <div>
```

Comment:
This `div` is unnecessary. It'd be good to take it out to reduce items in the tree.